ft_crossfrequencyanalysis.m

```diff
@@ -111,7 +111,8 @@
   docrosschan   = true;
   cfg.chanlow   = ft_channelselection(cfg.chanlow, freqlow.label);
   cfg.chanhigh  = ft_channelselection(cfg.chanhigh, freqhigh.label);
-  labelcmb = ft_channelcombination({cfg.chanlow,cfg.chanhigh},union(freqlow.label, freqhigh.label),1,2);
+  labelcmb = ft_channelcombination({cfg.chanlow,cfg.chanhigh},union(freqlow.label, freqhigh.label),1,0);
+  labelcmb(ismember(labelcmb(:,1),cfg.chanhigh)&strcmp(labelcmb(:,1),labelcmb(:,2)),:) = [];
```

Comment:
Hi @schoffelen, is there a reason why you changed the code here? It is not what I intended. Can you change it (line 114-115) back?
In case when `isequal(cfg.chanhigh,cfg.chanlow)`, labelcmb should list all combinations with all directions and including auto.

Reply:
But what is the intended behavior if cfg.chanhigh ~= cfg.chanlow?

Reply:
If you have a specific hypothesis about connectivity and do not what to test the whole space but rather a subset of the locations only.

Reply:
E.g.: fronto-parieta PAC only

Reply:
In general, the intended behaviour is that `labalcmb` contains pairs of all items of chanlow to all items of chanhigh regardless, whether there is an overlap between the two.

Reply:
I understand that, but if you call ft_channelcombination with ({low high},labellist,1,2), (with low and high disjoint) then you will end up with n^2 pairs, which means that what was initially intended to be channels with the high frequency envelope also end up as being the candidate channels to deliver the 'low' phase signal. To me it seems that this is not intended, and might fail. I am happy to change it  back, no problem.

Reply:
Yes. But I would leave that decision to the user, whether (s)he specifies pairs with both directions (i.e. overlap between chanlow and chanhigh)..